### PR TITLE
updates for quick endpoint changes and riot trust_level

### PR DESCRIPTION
**Pull request by bradchiappetta**
Updates to support adding RIOT to quick endpoint
- updated quick command
- updated analyze command
- updated filter command

Updates to support trust_level for riot in formatter

### CODE REVIEW

**Comment by bradchiappetta** [comment]
@andrew-morris *bumping* please review when you have cycles so we can get this out to folks.  Thanks!

File: tests/cli/test_subcommand.py [hunk 1]
```diff
@@ -280,7 +285,32 @@ def test_noise_only(self, api_client, text, expected_output):
...
+        }
+
+    @pytest.mark.parametrize(
+        "text, expected_output",
+        [
+            ("<input_text>", "<output_text>"),
+            ("<input_text>", ("<chunk_1>\n", "<chunk_2>\n")),
+        ],
+    )
+    def test_riot_only(self, api_client, text, expected_output):
```
Comment (Obsecurus) [inline tests/cli/test_subcommand.py:300]: 👍🏼

File: src/greynoise/api/filter.py [hunk 2]
```diff
@@ -72,6 +83,21 @@ def all_ip_addresses_noisy(line):
                 for line_ip_address in line_ip_addresses
             )
 
+        def all_ip_addresses_riot(line):
```
Comment (Obsecurus) [inline src/greynoise/api/filter.py:86]: 👍🏼

**Review by Obsecurus — APPROVED** [review]
Nice work!

**Pull request merged**